web/src/components/ThemeSettings/ThemeSettings.js

```diff
@@ -0,0 +1,76 @@
+import { Toaster } from '@redwoodjs/web/toast'
+const ThemeSettings = ({ data, update, loading }) => {
+  const handleClick = (e) => {
```

Comment:
This function should take the theme selection as a parameter then we can remove all if/else logic.

Also, since the only thing we are doing here is selecting the theme, let's rename the function.

```suggestion
  const selectTheme = (themeSelection) => {
```